src/test/java/uk/gov/hmcts/reform/blobrouter/tasks/processors/BlobProcessorTest.java

```diff
@@ -285,7 +286,8 @@ private void verifyNewEnvelopeHasBeenCreated() {
         verify(envelopeService).createNewEnvelope(
             blobClient.getContainerName(),
             blobClient.getBlobName(),
-            blobClient.getProperties().getCreationTime().toInstant()
+            blobClient.getProperties().getCreationTime().toInstant(),
+            blobClient.getProperties().getBlobSize()
```

Comment:
which one is correct `blobClient.getProperties().getBlobSize()` or `b.getProperties().getContentLength()`

Reply:
I assume they are interchangeable:

- in DuplicateFinder BlobItem class is used (`b.getProperties().getContentLength()`)
- in BlobProcessor BlobClient class is used (`blobClient.getProperties().getBlobSize()`)

`blobClient` is created in our ContainerProcessor class like this:
`containerClient.getBlobClient(blobItem.getName())`

I will rename `b` to `blobItem` in `DuplicateFinder` for better clarity.